source/extensions/filters/network/rocketmq_proxy/BUILD

```diff
@@ -0,0 +1,168 @@
+licenses(["notice"])  # Apache 2
+
+load(
+    "//bazel:envoy_build_system.bzl",
+    "envoy_cc_extension",
+    "envoy_cc_library",
+    "envoy_package",
+)
+
+envoy_package()
+
+envoy_cc_library(
+    name = "stats_interface",
+    hdrs = ["stats.h"],
+    deps = [
+        "//include/envoy/stats:stats_interface",
+        "//include/envoy/stats:stats_macros",
+    ],
+)
+
+envoy_cc_library(
+    name = "rocketmq_interface",
+    hdrs = [
+        "message_queue.h",
+        "topic_route.h",
+    ],
+    external_deps = ["rapidjson"],
```

Comment:
Let's not depend on rapidjson https://github.com/envoyproxy/envoy/issues/4705

Reply:
I also had the same ideas, 😄

Reply:
OK